src/Core/Asset/Camera.cpp

```diff
@@ -164,14 +133,17 @@ void Camera::fitZRange( const Core::Aabb& aabb ) {
     adaptRange( maxAabb[0], minAabb[1], maxAabb[2] );
     adaptRange( maxAabb[0], minAabb[1], minAabb[2] );
 
-    // ensure a minimum depth range
-    Scalar range = ( m_zFar - m_zNear ) / 100_ra;
-    m_zNear      = std::max( range, m_zNear - range );
-    m_zFar       = std::max( 2_ra * range, m_zFar + range );
+    if ( m_zNear < m_minZNear ) { m_zNear = m_minZNear; }
+    if ( ( m_zFar - m_zNear ) < m_minZRange ) { m_zFar += m_minZRange - ( m_zFar - m_zNear ); }
```

Comment:
```suggestion
    if ( ( m_zFar - m_zNear ) < m_minZRange ) { m_zFar =  m_zNear + m_minZRange; }
```

Reply:
right, we fixed this one on another branch, I'll do a cherry pick for this PR.,